bindings/java/src/main/native/request.cc

```diff
@@ -2,17 +2,16 @@
  * Copyright (C) Mellanox Technologies Ltd. 2019. ALL RIGHTS RESERVED.
  * See file LICENSE for terms.
  */
-
+#include "jucx_common_def.h"
 #include "org_openucx_jucx_ucp_UcpRequest.h"
 
-#include <ucp/api/ucp.h>
-#include <ucs/type/status.h>
 
-JNIEXPORT jboolean JNICALL
-Java_org_openucx_jucx_ucp_UcpRequest_isCompletedNative(JNIEnv *env, jclass cls,
-                                                       jlong ucp_req_ptr)
+JNIEXPORT void JNICALL
+Java_org_openucx_jucx_ucp_UcpRequest_updateRequestStatus(JNIEnv *env,
+                                                         jobject jucx_request, jlong ucp_req_ptr)
```

Comment:
wrap lines

Reply:
It's formatted by clang:
```
-JNIEXPORT void JNICALL
-Java_org_openucx_jucx_ucp_UcpRequest_updateRequestStatus(JNIEnv *env,
-                                                         jobject jucx_request, jlong ucp_req_ptr)
+JNIEXPORT void JNICALL Java_org_openucx_jucx_ucp_UcpRequest_updateRequestStatus(
+    JNIEnv *env, jobject jucx_request, jlong ucp_req_ptr)
 {
     jucx_request_update_status(env, jucx_request,
                                ucp_request_check_status((void *)ucp_req_ptr));
```
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=11689&view=logs&jobId=cc064a77-22b5-56bf-ecc0-70b5fe764261&j=cc064a77-22b5-56bf-ecc0-70b5fe764261&t=aedfd754-44a6-53e6-6843-8659d800fee2

Let me fix, so it doesn't touch java